app/assets/scripts/components/share.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 import React from 'react';
 import url from 'url';
+import hideOnBlur from 'react-click-outside';
```

Comment:
@dereklieu I'm happy to add a delay, but I'm not sure I understand the rational in this case. Is it to avoid the possibility that a mobile user would mistakenly close the dropdown by touching the wrong part of the screen? If so, [the fix](https://github.com/map-egypt/map-egypt.github.io/pull/273/files#diff-cd44a66fbd29a13df599511fec2c8fc1R31) I just pushed won't help. How would you approach this?